proposals/remove-prefix-from-config.md

```diff
@@ -0,0 +1,180 @@
+1. Customers do not have the flexibility to use the resource name they wish to have unless they set the prefixes to empty string.
+
+The 3 prefixes that are used in the configuration are:
+* **compose-project-name-prefix** - name used for create task definition (<compose_project_name_prefix> + <project_name>)
+* **compose-service-name-prefix** - name used for create service (<compose_service_name_prefix> + <project_name>)
+* **cfn-stack-name-prefix** - name used for creating CFN stacks (<cfn_stack_name_prefix> + <cluster_name>)
+
+# Solution
+When users run `ecs-cli configure`, the prefixes will be removed from the config.
+```yaml
+default: my_config
+clusters:
+  my_config:
+  - cluster: demo
+  - region: us-west-2
+```
+
+When CLI detects that there is a different `cfn-stack-name` configured, the `cfn-stack-name` will be saved in the config, otherwise will default to `amazon-ecs-cli-setup-<cluster_name>`, similar to `compose-service-name-prefix` will be saved in the config if present.
+```yaml
+default: my_config
+clusters:
+  my_config:
+  - cluster: demo
+  - region: us-west-2
+  - cfn-stack-name: amazon-ecs-cli-demo
+  - compose-service-name-prefix: ecscompose-service-
+```
+
+# Workflow
+Assume project name is "myApp"
+### Existing customer first run
+1. Config help
+   ```
+   $ ecs-cli configure --help
+   --cluster value, -c value            Specifies the ECS cluster name to use. If the cluster does not exist, it is created when you try to add resources to it with the ecs-cli up command. [$ECS_CLUSTER]
+   --region value, -r value             Specifies the AWS region to use. If the AWS_REGION environment variable is set when ecs-cli configure is run, then the AWS region is set to the value of that environment variable. [$AWS_REGION]
+   --cfn-stack-name value               [Optional] Specifies the name of AWS CloudFormation stack created on ecs-cli up. (default: "amazon-ecs-cli-setup-<cluster-name>")
+   --migrate                            [Optional] Moves your old configuration to new configuration
+   --force-migrate                      [Optional] Force move your old configuration to new configuration
+   --compose-service-name-prefix value  [Deprecated] Specifies the prefix added to an ECS service created from a compose file. Format <prefix><project-name>. (default to empty)
+   ```
+
+1. Migrate config with prompts
+   ```
+   $ ecs-cli configure --migrate
+   Old config
+   -----------------------------------------------------
+   ~/.ecs/config
+   [ecs]
+   cluster = cli-demo
+   aws_profile =
+   region = us-west-2
+   aws_access_key_id  = *********
+   aws_secret_access_key  = *********
+   compose-project-name-prefix = ecscompose-
+   compose-service-name-prefix = ecscompose-service-
+   cfn-stack-name-prefix       = amazon-ecs-cli-setup-
+
+   New configs
+   -----------------------------------------------------
+   ~/.ecs/cluster.config
+   default: ecs
+   clusters:
+     ecs:
```

Comment:
recommend using **cluster-configurations:** instead of **clusters:**.     Otherwise the key below **ecs:** is a cluster?  That's going to confuse a lot of people.